Program.cs

```diff
@@ -1,4 +1,36 @@
 ﻿using System;
-Console.WriteLine ("Hello, World!");
-
-   
\ No newline at end of file
+for (; ; ) {
+   Console.Write ("Enter a password: ");
+   string password = Console.ReadLine ();
+   bool hasLength = password.Length >= 6;
+   bool hasDigit = false;
+   bool hasLower = false;
+   bool hasUpper = false;
+   bool hasSpecial = false;
+   foreach (char c in password) {
+      if (char.IsDigit (c))

```

Comment:
Also check whether 'hasDigit' is true or false. Because if it is already 'true' again we don't want to write it.
Also write this for all If conditions.